ext/node/ops/http.rs

```diff
@@ -0,0 +1,125 @@
+use deno_core::error::bad_resource;
+use deno_core::error::type_error;
+use deno_core::error::AnyError;
+use deno_core::futures::StreamExt;
+use deno_core::op;
+use deno_core::url::Url;
+use deno_core::AsyncRefCell;
+use deno_core::ByteString;
+use deno_core::CancelFuture;
+use deno_core::CancelHandle;
+use deno_core::OpState;
+use deno_core::ResourceId;
+use deno_core::ZeroCopyBuf;
+use deno_fetch::get_or_create_client_from_state;
+use deno_fetch::BytesStream;
+use deno_fetch::FetchCancelHandle;
+use deno_fetch::FetchRequestBodyResource;
+use deno_fetch::FetchRequestResource;
+use deno_fetch::FetchResponse;
+use deno_fetch::FetchReturn;
+use deno_fetch::HttpClientResource;
+use deno_fetch::MpscByteStream;
+use reqwest::header::HeaderMap;
+use reqwest::header::HeaderName;
+use reqwest::header::HeaderValue;
+use reqwest::header::CONTENT_LENGTH;
+use reqwest::Body;
+use reqwest::Method;
+use std::cell::RefCell;
+use std::rc::Rc;
+
+#[op]
+pub fn op_node_http_request(
```

Comment:
Maybe add a comment to duplicate with op_fetch from https://github.com/denoland/deno/blob/0ccfccdcd271cb75f7453c70c6c3f4cadd7f3858/ext/fetch/lib.rs#L215

Reply:
We'll deduplicate in a follow up PR